starknet_devnet/blueprints/base.py

```diff
@@ -126,25 +126,33 @@ def load():
 
 
 @base.route("/increase_time", methods=["POST"])
-def increase_time():
-    """Increases the block timestamp offset"""
+async def increase_time():
+    """Increases the block timestamp offset and generate block"""
```

Comment:
```suggestion
    """Increases the block timestamp offset and generates a block"""
```

Reply:
fixed in bedd89c410df8d46c1a98138aa4cdbb9b53f69eb